pkg/pillar/cmd/msrv/patchenvelopes.go

```diff
@@ -87,8 +96,9 @@ func NewPatchEnvelopes(log *base.LogObject, ps *pubsub.PubSub) *PatchEnvelopes {
 		completedVolumes:  generics.NewLockedMap[uuid.UUID, types.VolumeStatus](),
 		contentTreeStatus: generics.NewLockedMap[uuid.UUID, types.ContentTreeStatus](),
 
-		log:    log,
-		pubSub: ps,
+		log:    msrv.Log,
+		pubSub: msrv.PubSub,
+		msrv:   msrv,
```

Comment:
why does PatchEnvelope is dependent on metadata server? I though of PatchEnvelopes as something that server is sharing via it's endpoints

Reply:
well, the actions in the cipherblob, I do need to access a number of existing functions, from msrv functions.